bokehjs/make/tasks/pack.ts

```diff
@@ -10,16 +10,17 @@ function npm_pack() {
     fs.mkdirSync(dist, {recursive: true})
   }
 
-  const npm = process.platform != "win32" ? "npm" : "npm.cmd"
-  const {status, stdout, stderr} = cp.spawnSync(npm, ["pack", `--pack-destination=${dist}`], {stdio: "pipe", encoding: "utf-8"})
+  const is_windows = process.platform == "win32"
+  const npm = !is_windows ? "npm" : "npm.cmd"
```

Comment:
easier to follow an affirmative condition
```suggestion
  const npm = is_windows ? "npm.cmd" : "npm"
```